### PR TITLE
Rework the handler includes, add travis testing, depend on ephemeris and conda dependency installation support

### PR DESCRIPTION
**Pull request by mvdbeek**
- Include the complete restart_galaxy.yml as handler, previously ansible has just silently picked the restart galaxy task instead of running through all the tasks actually required to restart galaxy
- Explicitly check for the correct ansible version and abort if this is not satisfied
- Add travis testing (Spin up bjoern's container, and install tools from the [tool_list.yaml.sample](https://github.com/galaxyproject/ansible-galaxy-tools/blob/master/files/tool_list.yaml.sample) file
- Depend on [ephemeris](https://github.com/galaxyproject/ephemeris) for providing the tool install script. Through ephemeris we now support installing conda dependencies into galaxy instances running 16.07 or newer.

This is still blocked by https://github.com/ansible/ansible/issues/15915. The travis tests are running with ansible 2.1.2.0-rc1.

### CODE REVIEW

**Comment by mvdbeek** [comment]
Well, this is still stuck with https://github.com/ansible/ansible/issues/15915, but could someone activate travis testing for this repo? @afgane @jmchilton @bgruening ?

**Comment by jmchilton** [comment]
Travis has been enabled for this repo - thanks for the heads up and thanks for the awesome tests!

File: tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ---
+- fail: msg="Installed ansible version {{ ansible_version.full }}, but ansible version >= 2.1.1.0 required"
+  when: "{{ ansible_version.full | version_compare('2.1.1.0', '<') }}"
```
Comment (bgruening) [inline tasks/main.yml]: `min_ansible_version`?

**Comment by bgruening** [comment]
@mvdbeek shouldn't this test fail if we are waiting for an ansible fix?
Nice work!

**Comment by mvdbeek** [comment]
I'm cloning from ansible's dev repo in the .travis.yml file, that's why the tests pass.

File: tasks/main.yml (hunk 1 above)
Comment (mvdbeek) [inline tasks/main.yml]: I think this only applies when using ansible-galaxy to manage roles.
We've run into sooooo much trouble with differing ansible versions that silently fail due to some version-specific bugs/subtle difference that I prefer to fail explicitly.

**Comment by mvdbeek** [comment]
I think I'm going mad. Now that the ansible folks are releasing the named include fix, they've broken the path to the included files. And fixed it only in their 2.2 dev branch. arg. ahhhh

**Comment by bgruening** [comment]
Same release manager as the conda folks? ;)

**Comment by jmchilton** [comment]
Cause we'd never release a Galaxy without bugs :wink:

**Comment by mvdbeek** [comment]
> Cause we'd never release a Galaxy without bugs 😉

We just have usability issues, that's not the same 😆

**Comment by mvdbeek** [comment]
Found a workaround, yay!

**Comment by afgane** [comment]
Tested and working nicely.

**Pull request merged**